lang/chicken-scheme/Makefile

```diff
@@ -40,24 +50,98 @@ define Package/chicken-scheme-interpreter/description
   For more information, please refer to the Chicken Scheme website at https://call-cc.org.
 endef
 
-MAKE_FLAGS += PLATFORM=linux C_COMPILER=$(TARGET_CC) LINKER=$(TARGET_CC) PREFIX=/usr C_COMPILER_OPTIMIZATION_OPTIONS="$(TARGET_CFLAGS)"
+
+# csc calls gcc with "-lm -ldl", but "-lm" doesn't work with musl,
+# so we set LIBRARIES=-lm
+MAKE_FLAGS += PLATFORM=linux C_COMPILER=$(TARGET_CC) LINKER=$(TARGET_CC) PREFIX=/usr C_COMPILER_OPTIMIZATION_OPTIONS="$(TARGET_CFLAGS)" LIBRARIES=-lm
 
 # not installed:
 # - csc and chicken, the compiler
 # - the include dir (only useful with the compiler)
-# - install, uninstall, status, and chicken-do, which deal with modules (installation of more modules depends on the compiler)
+# - install, uninstall, status, and chicken-do, which deal with modules
+#   (installation of more modules depends on the compiler)
 # - profiler
 # - feathers, the debugger
 # - libchicken.a, the static library
 define Package/chicken-scheme-interpreter/install
 	$(INSTALL_DIR)  $(1)/usr/bin
-	$(INSTALL_DIR)  $(1)/usr/lib/chicken/$(ABI_VERSION)
```

Comment:
This whole section, why?

Reply:
Thank you! Don't know why I ended up using `11` instead of `$(ABI_VERSION)`.